### PR TITLE
Add Steam

### PR DESCRIPTION
**Pull request by TallGuysFreeADMIN**

### CODE REVIEW

File: data/steam.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "id": "steam",
+  "name": "Steam",
+  "description": "Digital distribution platform by Valve ",
+  "categories": [],
+  "iconUrl": "https://i.imgur.com/IskAj8F.jpg",
+  "website": "https://store.steampowered.com",
+  "applicationUrl": "https://store.steampowered.com/",
+  "countryHQ": "US",
+  "gdprReadyStatus": "ready",
```
Comment (Gdewilde) [inline data/steam.json]: Did you read an official about this? If yes, please add this to the articles. If not I would choose inProgress.

Comment (TallGuysFreeADMIN) [reply data/steam.json]: It's in their privacy policy ............... https://store.steampowered.com/privacy_agreement/ :/

Comment (Gdewilde) [reply data/steam.json]: ok

**Review by Gdewilde — APPROVED** [review]

**Comment by TallGuysFreeADMIN** [comment]
not a fan of the changes in https://github.com/privacyradius/gdpr-tracker/pull/129/commits/ba6b3efb9ac614e6c6393c7c6b574613fe097357. 

"Steam Store" is not the name of the website. The website is for Steam and privacy policy talks about Steam (valve, actually).
https://partner.steamgames.com/doc/marketing/branding/attribution
"©2017 Valve Corporation. **Steam** and the Steam logo are **trademarks** and/or registered trademarks of Valve Corporation in the U.S. and/or other countries."

**Comment by Gdewilde** [comment]
Ok, I see your point. You can revert the changes if you want.

**Pull request merged**